nestkernel/io_manager.cpp

```diff
@@ -309,9 +309,9 @@ IOManager::register_recording_backends_()
   recording_backends_.insert( std::make_pair( "ascii", new RecordingBackendASCII() ) );
   recording_backends_.insert( std::make_pair( "memory", new RecordingBackendMemory() ) );
   recording_backends_.insert( std::make_pair( "screen", new RecordingBackendScreen() ) );
-#ifdef HAVE_MPI
+#ifdef HAVE_RECORDINGBACKEND_ARBOR
   recording_backends_.insert( std::make_pair( "arbor", new RecordingBackendArbor() ) );
-#endif
+#endif // HAVE_RECORDINGBACKEND_ARBOR
```

Comment:
```suggestion
#endif
```
I think the conditional block is short enough to be understood without the comment, which only adds clutter in my view.